lib/casbin-ruby/core_enforcer.rb

```diff
@@ -9,46 +9,47 @@
 require 'casbin-ruby/util'
 require 'casbin-ruby/util/builtin_operators'
 require 'casbin-ruby/util/evaluator'
-
-require 'logger'
+require 'casbin-ruby/logger'
 
 module Casbin
   # CoreEnforcer defines the core functionality of an enforcer.
   # get_attr/set_attr methods is ported from Python as attr/attr=
   class CoreEnforcer
-    def initialize(model = nil, adapter = nil, logger: Logger.new($stdout))
+    def initialize(model = nil, adapter = nil, logger: ::Logger.new($stdout), level: 'error')
+      Logger.register(logger, level)
```

Comment:
And
```suggestion
      Logger.register(logger)
```

So we can simplify the `Logger` module.

Reply:
Аnd another question, what if we have two enforcers?